Orange/base.py

```diff
@@ -22,14 +62,15 @@ class Learner:
     learner_adequacy_err_msg = ''
 
     def __init__(self, preprocessors=None):
-        if preprocessors is None:
-            preprocessors = type(self).preprocessors
-        self.preprocessors = list(preprocessors)
+        self.use_default_preprocessors = False
+        if isinstance(preprocessors, Iterable):
+            self.preprocessors = preprocessors
```

Comment:
By doing this, I also realised most of my tests concerning preprocessors actually belonged to the learner tests. I've moved them to `Orange/tests/test_base.py` since I was unable to find any other tests regarding the `Learner` class.